packages/dma-library/src/types/ajna/ajna-position.ts

```diff
@@ -125,6 +132,17 @@ export class AjnaPosition implements IAjnaPosition {
       .minus(this.debtAmount.times(this.quotePrice))
   }
 
+  pnl(
+    cumulativeDepositUSD: BigNumber,
+    cumulativeWithdrawnUSD: BigNumber,
+    cumulativeFeesUSD: BigNumber,
+  ): BigNumber {
+    return this.netValue
+      .minus(cumulativeDepositUSD)
+      .plus(cumulativeWithdrawnUSD)
+      .minus(cumulativeFeesUSD)
+  }
```

Comment:
```export function calculatePNL(events: VaultEvent[], currentNetValueUSD: BigNumber) {
  const cumulativeDepositUSD = events.reduce(getCumulativeDepositUSD, zero)
  const cumulativeWithdrawnUSD = events.reduce(getCumulativeWithdrawnUSD, zero)
  const cumulativeFeesUSD = events.reduce(getCumulativeFeesUSD, zero)

  if (cumulativeDepositUSD.isZero()) {
    return zero
  }

  return cumulativeWithdrawnUSD
    .plus(currentNetValueUSD)
    .minus(cumulativeFeesUSD)
    .minus(cumulativeDepositUSD)
    .div(cumulativeDepositUSD)
}```

We have slightly different formula in borrow. Can we check which one is correct?

Reply:
My bad. Copied the wrong one like a muppet 🤦‍♂️

Reply:
🫂